pages/k8s/storage.md

```diff
@@ -88,7 +93,8 @@ juju run-action ceph-mon/0 delete-pool pool-name=xfs-pool --wait
 juju run-action ceph-mon/0 delete-pool pool-name=ext4-pool --wait
 ```
 
-Then recreate them:
+Then recreate them, using the options listed from the `list-actions` command ran
+earlier.  For example:
```

Comment:
for example, no options! :-P

I hate to keep poking this on you, but could you change the run-action to use a different option like more replicas or something?

Reply:
No problem, measure twice; cut once and all that.  I was assuming that was explicit with what was above, but I'll add some changed args.

Reply:
I like it, it gives a user an idea of what they might not like about the defaults and shows them how to change the values around.